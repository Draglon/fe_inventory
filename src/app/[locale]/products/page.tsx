export { default } from "@/views/Products";
