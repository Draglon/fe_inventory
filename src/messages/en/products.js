const products = {
  "Products": {
    "layout": {
      "title": "Products page",
      "description": "Products page - description",
    },
  },
};

export default products;