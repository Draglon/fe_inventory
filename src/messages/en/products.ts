const products = {
  "Products": {
    "layout": {
      "title": "Products page",
      "description": "Products page - description",
    },
    "title": "Products",
    "filter": {
      "type": {
        "label": "Type:",
        "option": {
          "all": "All",
          "phone": "Phone",
          "monitor": "Monitor"
        }
      },
      "specification": {
        "label": "Specification:",
        "option": {
          "all": "All",
          "phone": "Phone",
          "monitor": "Monitor"
        }
      },
    },
    "guarantee": {
      "from": "from",
      "to": "to"
    }
  },
};

export default products;