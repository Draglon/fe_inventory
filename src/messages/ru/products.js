const products = {
  "Products": {
    "layout": {
      "title": "Продукты",
      "description": "Продукты - описание",
    },
    "title": "Продукты",
  },
};

export default products;