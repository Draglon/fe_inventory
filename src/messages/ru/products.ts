const products = {
  "Products": {
    "layout": {
      "title": "Продукты",
      "description": "Продукты - описание",
    },
    "title": "Продукты",
    "filter": {
       "type": {
        "label": "Тип:",
        "option": {
          "all": "Все",
          "phone": "Телефон",
          "monitor": "Монитор"
        }
      },
      "specification": {
        "label": "Спецификация:",
        "option": {
          "all": "Все",
          "phone": "Телефон",
          "monitor": "Монитор"
        }
      },
    },
    "guarantee": {
      "from": "с",
      "to": "по"
    }
  },
};

export default products;