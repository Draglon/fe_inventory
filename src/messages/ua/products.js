const products = {
  "Products": {
    "layout": {
      "title": "Продукти",
      "description": "Продукти",
    },
    "title": "Продукти",
  },
};

export default products;