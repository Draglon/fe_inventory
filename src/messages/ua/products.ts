const products = {
  "Products": {
    "layout": {
      "title": "Продукти",
      "description": "Продукти",
    },
    "title": "Продукти",
    "filter": {
      "type": {
        "label": "Тип:",
        "option": {
          "all": "Всі",
          "phone": "Телефон",
          "monitor": "Монітор"
        }
      },
      "specification": {
        "label": "Специфікація:",
        "option": {
          "all": "Всі",
          "phone": "Телефон",
          "monitor": "Монітор"
        }
      },
    },
    "guarantee": {
      "from": "з",
      "to": "до"
    }
  },
};

export default products;