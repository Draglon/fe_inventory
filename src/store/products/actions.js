import { productsSlice } from "./reducer";

export const { fetchProducts, createProduct, deleteProduct } = productsSlice.actions;
