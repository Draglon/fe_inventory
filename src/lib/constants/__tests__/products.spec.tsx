import * as products from "../products";

it("products constants match snapshot", () => {
  expect(products).toMatchSnapshot();
});
