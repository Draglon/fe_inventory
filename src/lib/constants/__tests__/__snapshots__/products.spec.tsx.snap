// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`products constants match snapshot 1`] = `
{
  "PRODUCTS": [
    {
      "date": "2017-06-29 12:09:33",
      "fullName": null,
      "group": "Длинное предлинное длиннючее название группы",
      "guarantee": {
        "end": "2017-06-29 12:09:33",
        "start": "2017-06-29 12:09:33",
      },
      "id": "1",
      "isNew": 1,
      "order": 1,
      "parish": "Длинное предлинное длиннючее название прихода",
      "photo": {
        "blurDataURL": "data:image/png;base64,imagedata",
        "height": 40,
        "src": "/img.jpg",
        "width": 40,
      },
      "price": [
        {
          "isDefault": 0,
          "symbol": "USD",
          "value": 100,
        },
        {
          "isDefault": 1,
          "symbol": "UAH",
          "value": 2600,
        },
      ],
      "serialNumber": "SN-12.3456789",
      "specification": "Specification 1",
      "status": "В наличии",
      "title": "Product 1",
      "type": "Monitors",
    },
    {
      "date": "2017-06-29 12:09:33",
      "fullName": "John Smith",
      "group": "Длинное предлинное длиннючее название группы",
      "guarantee": {
        "end": "2017-06-29 12:09:33",
        "start": "2017-06-29 12:09:33",
      },
      "id": "2",
      "isNew": 0,
      "order": 1,
      "parish": "Длинное предлинное длиннючее название прихода",
      "photo": {
        "blurDataURL": "data:image/png;base64,imagedata",
        "height": 40,
        "src": "/img.jpg",
        "width": 40,
      },
      "price": [
        {
          "isDefault": 0,
          "symbol": "USD",
          "value": 100,
        },
        {
          "isDefault": 1,
          "symbol": "UAH",
          "value": 2600,
        },
      ],
      "serialNumber": "SN-12.3456789",
      "specification": "Specification 1",
      "status": "В ремонте",
      "title": "Інтерактивна панель Promethean ActivPanel LX 75 з модулем OPS-A",
      "type": "Monitors",
    },
    {
      "date": "2017-06-29 12:09:33",
      "fullName": null,
      "group": "Длинное предлинное длиннючее название группы",
      "guarantee": {
        "end": "2017-06-29 12:09:33",
        "start": "2017-06-29 12:09:33",
      },
      "id": "3",
      "isNew": 0,
      "order": 1,
      "parish": "Длинное предлинное длиннючее название прихода",
      "photo": {
        "blurDataURL": "data:image/png;base64,imagedata",
        "height": 40,
        "src": "/img.jpg",
        "width": 40,
      },
      "price": [
        {
          "isDefault": 0,
          "symbol": "USD",
          "value": 100,
        },
        {
          "isDefault": 1,
          "symbol": "UAH",
          "value": 2600,
        },
      ],
      "serialNumber": "SN-12.3456789",
      "specification": "Specification 1",
      "status": "В ремонте",
      "title": "Інтерактивна панель Promethean ActivPanel LX 75 з модулем OPS-A",
      "type": "Monitors",
    },
  ],
}
`;
